cactusbot/commands/magic/command.py

```diff
@@ -91,3 +117,30 @@ class Meta(Command):
         response = await self.api.command.update_count(command, action_string)
         if response.status == 200:
             return "Count updated."
+
+    @Command.command(role="moderator")
+    async def permission(self, command: r'?command', role=None):
```

Comment:
Didn't we want to use the `rename` syntax (in discussion on #249), rather than `permission`?

```
!command rename stuff things
!command rename test +test
```